### DISCUSSION TITLE
Session package problems

### DISCUSSION THREAD

**Discussion by mvyleteljr** [adonisjs/core#3384] [Help]
I ran `npm i @adonisjs/session` to install session package, and subsequently: `node ace configure @adonisjs/session` to configure it, but I am still getting a "Module not found error" for `'@ioc:Adonis/Addons/Session'`in the config file for session... 

As a result, session is not being shown as a property of `HttpContextContract`, so I cannot use the session package at all. Not sure where to go from here, and any help would be appreciated

**Comment by thetutlage** [comment]
Restart or reload your editor. It is very common for editors to not pick new types from `node_modules`
